module/gui/PackageDock.py

```diff
@@ -40,51 +59,214 @@ def slotDone(self):
             if not line:
                 continue
             lines.append(line)
-        self.emit(SIGNAL("done"), str(self.widget.nameInput.text()), lines, pw)
+        if not lines:
+            self.widget.slotMsgShow("<b>" + "&nbsp;" + "&nbsp;" + _("Error, no URLs given.") + "</b>")
+            return
+        queue = self.widget.destQueue.isChecked()
+        self.emit(SIGNAL("done"), unicode(self.widget.nameInput.text()), lines, queue, pw)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['SIGNAL' may be undefined, or defined from star imports: PyQt4.QtCore, PyQt4.QtGui](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=567903)